mem-15-S.xml

```diff
@@ -6859,7 +6859,7 @@ This verb is also used for the encryption of data.[3]</column>
       <column name="entry_name">Sub</column>
       <column name="part_of_speech">v:1,is</column>
       <column name="definition">be solid</column>
-      <column name="definition_de">fest, hart, solide</column>
+      <column name="definition_de">fest, hart, solide sein</column>
```

Comment:
better: "fest, solide sein"
(not every solid is hard)

Reply:
I'm not even sure that this adjectival sense of "be solid" has anything to do with solids (meaning the state of matter). To "be solid" could simply mean to be firm or reliable (in character), or not hollowed out (of a physical object). The slang meaning seems to suggest that it has to do with character. But in the absence of more information I guess this is fine.